recipes/ddocent/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set version = "2.6.0" %}
-{% set sha256 = "b985ea61fbf3e096b81f73c0afff563e11fbcca3731a09627d3b4de8013f9778" %}
+{% set sha256 = "872055abf47e36643904c8b47b7e9c9f92206f2e85b3d6701906b4781924e63b" %}
```

Comment:
Why is this? Is it a new version?

Reply:
It’s a re-release of this version.  There were some errors in a few supplemental scripts if run on OS X.  The program code itself didn't change, so I didn't want to bump the version number.